templates/components/form/form.jsx

```diff
@@ -224,11 +231,14 @@ var Form = React.createClass({
   /**
    * "owner" components call form.processFormData on us
    */
-  processFormData: function(e) {
-    e.preventDefault();
-    this.validate((error, data) => {
-      WebmakerActions.onFormValidation({err: error, user: !!error ? false : JSON.parse(JSON.stringify(this.state))});
-    });
+  processFormData: function(evt, clicked) {
+    evt.preventDefault();
+    let keyCode = evt.keyCode;
```

Comment:
Don't think you need to cache this var if it's just used once below. Am I missing something you're doing with it here?